manifests/config.pp

```diff
@@ -61,15 +61,23 @@
 
   # Handle env variables.
   $environment_variables = merge($default_env_variables, $rabbitmq::environment_variables)
+  case $::osfamily {
+    'FreeBSD': {
+      $etc_dir = '/usr/local/etc'
+    }
+    default: {
+      $etc_dir = '/etc'
+    }
```

Comment:
this can just be a selector, since we're assigning one value:
```puppet
$etc_dir = $::osfamily? {
  'FreeBSD' => '/usr/local/etc',
  default   => '/etc'
}
```